### ISSUE TITLE
Rename files only

### ISSUE THREAD

**Issue by gnaghi** [avalonv/reCBZ#22]
Some cbz archives has badly named files, which result in wrong order when reading on e-reader.
For example : 
1.jpg, 10.jpg, 11.jpg should be renamed as 0001.jpg, 0010.jpg, 0011.jpg.
But without recompressing the files, only by renaming them.

**Comment by BelardoA** [comment]
Are you suggesting renaming the files something like this?

```shell
├── /path/to/cbz/files
│   ├──  1.jpg  -->  0001.jpg
│   ├── 10.jpg  -->  0010.jpg
│   ├── 11.jpg  -->  0011.jpg
```

**Comment by gnaghi** [comment]
Yes. Some archive also have unneeded directory, so it would not only rename the files, but also remove useless directories.
